### PR TITLE
Reduce deployment targets by 3 versions

### PR DESCRIPTION
**Pull request by andrewse02**
### Motivation

Reduce deployment target by 3 versions to gain more adoption.

### Modifications

- Reduced deployment targets by 3 versions
- Modified `Converter#renderedRequestPath` to support iOS 13

### Result

Deployment target will be reduced by 3 versions.

### Resolves

- Resolves https://github.com/apple/swift-openapi-generator/issues/62

### Related PRs

- Paired with https://github.com/apple/swift-openapi-generator/pull/63

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ import PackageDescription
 let package = Package(
     name: "swift-openapi-runtime",
     platforms: [
-        .macOS(.v13), .iOS(.v16), .tvOS(.v16), .watchOS(.v9),
+        .macOS(.v10_15), .iOS(.v12), .tvOS(.v13), .watchOS(.v6),
```
Comment (czechboy0) [inline Package.swift:20]:
```suggestion
        .macOS(.v10_15), .iOS(.v13), .tvOS(.v13), .watchOS(.v6),
```

Let's keep the OS versions aligned.

File: Sources/OpenAPIRuntime/Conversion/Converter+Client.swift [hunk 2]
```diff
@@ -22,11 +22,21 @@ extension Converter {
     ) throws -> String {
         var renderedString = template
         for parameter in parameters {
-            renderedString.replace(
-                "{}",
-                with: parameter.description,
-                maxReplacements: 1
-            )
+            if #available(iOS 16.0, macOS 13.0, *) {
```
Comment (czechboy0) [inline Sources/OpenAPIRuntime/Conversion/Converter+Client.swift:25]: Please check aligned watchOS and tvOS versions as well.

**Comment by czechboy0** [comment]
@swift-server-bot add to allowlist

**Review by czechboy0 — APPROVED** [review]
Thanks 🙏

**Review by simonjbeaumont — APPROVED** [review]
Thanks @andrewse02 for working on this.

**Pull request merged**